backends/ebpf/ebpfParser.cpp

```diff
@@ -293,6 +295,23 @@ void StateTranslationVisitor::compileExtractField(const IR::Expression *expr, cs
         builder->append(")");
         builder->endOfStatement(true);
     } else {
+        if (program->options.arch == "psa" && widthToExtract % 8 != 0) {
```

Comment:
yes, in my implementation I never figured out how all this stuff should be handled.